tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot

```diff
@@ -18,15 +20,28 @@ JupyterHub Spawner Is Visible
 Select Notebook Image
    [Documentation]  Selects a notebook image based on a partial match of ${notebook_image} argument
    [Arguments]  ${notebook_image}
-   Wait Until Element Is Visible  xpath:/html/body/div[1]/form/div/div/div[2]/div[2]/div[1]
+   ${ver} =  Get RHODS version
```

Comment:
This two lines could be condensed in one like this:
```
Before:
 ${ver} =  Get RHODS version
 ${version-check} =  GTE  ${ver}  1.3.0
 IF  ${version-check}==True

After:
${version-check} = RHODS Version Is Greater Or Equal Than  1.3.0
IF  ${version-check}==True
```

Reply:
Implemented in new commit